drv/sprot-api/src/lib.rs

```diff
@@ -71,173 +273,73 @@ pub enum SprotError {
     BadMessageType,
     /// Transfer size is outside of maximum and minimum lenghts for message type.
     BadMessageLength,
-    /// Error from Spi
-    SpiServerError,
-    /// Message is too large
-    Oversize,
-    /// Tx buffer is unexpectedly not Idle.
-    TxNotIdle,
-
-    // SPI-related errors
-    SpiServerLockError,
-    SpiServerWritePart1Error,
-    SpiServerWritePart2Error,
-    SpiServerReleaseError,
-    SpiServerReadPart1Error,
-    SpiServerReadPart2Error,
+    // We cannot assert chip select
     CannotAssertCSn,
```

Comment:
...why would we not be able to assert CS?

Reply:
Looks like I only use it in one place: https://github.com/oxidecomputer/hubris/pull/1279/files/8ff567f9af1aa58ca16496341b93455b5e935924#diff-32036439343487dbfb550235d8d3ce0a3a92e8d6808dc98d3e72d10d2e9d9c89R374